src/js/sensor_helpers.js

```diff
@@ -72,10 +76,17 @@ export function sensor_status(sensors_detected) {
 
     if (have_sensor(sensors_detected, "gps")) {
         $(".gps", eSensorStatus).addClass("on");
-        $(".gpsicon", eSensorStatus).addClass("active");
+        if (gps_fix_state > 0) {
```

Comment:
```suggestion
        if (gps_fix_state) {
```
should work as boolean